commons-component-common/src/main/java/org/exoplatform/commons/utils/HTMLSanitizer.java

```diff
@@ -131,6 +131,10 @@
                                                                                                                                 .allowStandardUrlProtocols()
                                                                                                                                 .allowAttributes("nohref")
                                                                                                                                 .onElements("a")
+                                                                                                                                .allowAttributes("target")
+                                                                                                                                .matching(true, "_blank")
+                                                                                                                                .onElements("a")
+                                                                                                                                .requireRelNofollowOnLinks()
```

Comment:
We need to add the rel="noopener" as suggested in https://github.com/OWASP/java-html-sanitizer/issues/147
Check https://github.com/OWASP/java-html-sanitizer/issues/147

Reply:
the `requireRelNofollowOnLinks` function should add nofollow, noopener and noreferrer yet this is not available in the current version of the owasp html sanitze dependancy 
I created this PR to update the dependancy : https://github.com/Meeds-io/maven-depmgt-pom/pull/61

Reply:
build and unit test will be okay once the PR of updating the sanitizer dependency merged